src/libmongoc/tests/json/crud/unified/bulkWrite-deleteMany-hint-unacknowledged.json

```diff
@@ -0,0 +1,269 @@
+{
+  "description": "bulkWrite-deleteMany-hint-unacknowledged",
```

Comment:
These `hint-unacknowledged` tests pertain to [CDRIVER-4127](https://jira.mongodb.org/browse/CDRIVER-4127), which is sitting in the backlog.

In the past, I think "sync" commits only pulled in the subset of test files relevant to the DRIVERS ticket vs. updating _all_ files in a directory to the target specifications commit. That seems like an easy way for spec tests to fall through the cracks, so I think it'd be preferable to keep these here.

Since we have the ability to skip tests, I see no reason to start syncing _everything_ in a directory and adding necessary skips and TODO comments as needed (as I did in `runner.c` for change stream spec tests).

These tests already seem to pass, but whenever [CDRIVER-4127](https://jira.mongodb.org/browse/CDRIVER-4127) gets scheduled we can double-check and resolve it accordingly if there's truly no work to be done.